sdcm/results_analyze.py

```diff
@@ -303,8 +305,7 @@ def check_regression(self, test_id, is_gce=False):
                        'hits.hits._source.results.stats_average',
                        'hits.hits._source.results.stats_total',
                        'hits.hits._source.versions']
-        tests_filtered = self._es.search(index=self._index, doc_type=test_type, q=query, filter_path=filter_path,
-                                         size=self._limit)
+        tests_filtered = self._es.search(index=self._es_index, q=query, filter_path=filter_path, size=self._limit)
```

Comment:
this search will return all the tests in the index, but we should filter certain test types only.
Yuo need to add the test_name to the query

Reply:
@bentsi ?